docs/source/configuration.rst

```diff
@@ -478,25 +478,23 @@ Input/output configuration options
     [blast]
     queries = /home/${USER}/blast-queries.tar.gz
 
-.. _elb_results_bucket:
+.. _elb_results:
 
-``Results bucket`` 
-^^^^^^^^^^^^^^^^^^
+``Results`` 
+^^^^^^^^^^^
 
-    GCS or AWS S3 bucket URI where to save the output from ElasticBLAST. 
+    GCS or AWS S3 bucket URI where to save the results from ElasticBLAST. 
```

Comment:
What was wrong with "results"?

Reply:
Actually, it used to be output, now it reads results (for consistency with the parameter name).